_sass/main.scss

```diff
@@ -442,14 +442,14 @@ table code {
   color: $yellow;
 }
 
-.blog-page, .post-page, .getting-started-page, .documentation, .OSD-2022-page, .hacktoberfest-page, .contribute-page, .guides-page, .setup-elytron {
+.blog-page, .post-page, .getting-started-page, .documentation, .OSD-2023-page, .contribute.page, .hacktoberfest-page, .guides-page, .setup-elytron {
```

Comment:
I think this can be just .OSD-page, right?

Reply:
fixed!